### PR TITLE
Add send transaction to sdk

### PR DESCRIPTION
**Pull request by jjsloboda**
Adds a version of `sendTransaction` call that supports the `validatorAcls` parameter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -800,6 +801,23 @@ const jitoApiUrl = 'https://mainnet.block-engine.jito.wtf/api/v1/bundles';
...
 
+### sendTransaction()
+This method behaves the same way as the standard `sendTransaction`, but adds support for `validatorAcls` JSON-based allow/denylists.
+
+```ts
+try {
+    const response = await helius.rpc.sendTransaction(
+        transaction,
+        {
+            validatorAcls: ['https://helius-docs.s3.us-east-2.amazonaws.com/sfdp_rejects.json'],
```
Comment (NicolasPennie) [inline README.md:812]: Should make the rejects an importable constant.

File: README.md [hunk 2]
```diff
@@ -800,6 +801,23 @@ const jitoApiUrl = 'https://mainnet.block-engine.jito.wtf/api/v1/bundles';
...
+            skipPreflight: true,
+        },
+    );
+} catch (error) {
+    console.log(error);
```
Comment (NicolasPennie) [inline README.md:817]: I know this is an example but use console.error

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "helius-sdk",
-  "version": "1.3.6",
+  "version": "1.3.7",
```
Comment (NicolasPennie) [inline package.json:3]: Technically 1.4.0 since this adds a new feature

File: src/RpcClient.ts [hunk 4]
```diff
@@ -1060,4 +1062,48 @@ export class RpcClient {
...
+   * @returns {Promise<TransactionSignature>} - The transaction signature
+   */
+  async sendTransaction(
+    transaction: Transaction | VersionedTransaction,
+    options: HeliusSendOptions = {
+      skipPreflight: true,
+    }
+  ): Promise<TransactionSignature> {
+    let rawTransaction: string;
+    if (transaction instanceof VersionedTransaction) {
```
Comment (NicolasPennie) [inline src/RpcClient.ts:1079]: Why do we need to do this?

File: src/RpcClient.ts [hunk 5]
```diff
@@ -1060,4 +1062,48 @@ export class RpcClient {
...
+      rawTransaction = Buffer.from(transaction.serialize()).toString('base64');
+    } else {
+      rawTransaction = transaction.serialize().toString('base64');
+    }
+
+    try {
+      const url = `${this.connection.rpcEndpoint}`;
+      const response = await axios.post(
```
Comment (NicolasPennie) [inline src/RpcClient.ts:1087]: Can we pass the values to web3js instead of re-implementing it?

File: src/RpcClient.ts [hunk 6]
```diff
@@ -1060,4 +1062,48 @@ export class RpcClient {
...
+          id: this.id,
+          method: 'sendTransaction',
+          params: [rawTransaction, {encoding: 'base64', ...options}],
+        },
+        {
+          headers: { 'Content-Type': 'application/json' },
+        }
+      );
+
+      if (response.data.error) {
```
Comment (NicolasPennie) [inline src/RpcClient.ts:1100]: Does this follow the same pattern as web3js?

File: src/RpcClient.ts (hunk 4 above)
Comment (jjsloboda) [reply src/RpcClient.ts:1079]: in web3js, `sendTransaction` has an overloaded version that accepts a `VersionedTransaction`, and a deprecated version that accepts `Transaction`. I wanted to make our version more user-friendly to anyone who is still using the deprecated `Transaction` interface. I can remove support for `Transaction` and only leave `VersionedTransaction` if you prefer

File: src/RpcClient.ts (hunk 6 above)
Comment (jjsloboda) [reply src/RpcClient.ts:1100]: at a high level, yes: throwing an error if there is an error returned in the RPC response
but specifically, I targeted it to match the rest of the helius SDK. it throws an Error object whereas the web3js version throws a `SendTransactionError` defined here: https://solana-labs.github.io/solana-web3.js/classes/SendTransactionError.html

**Review by NicolasPennie — APPROVED** [review]

**Pull request merged**